opentech/apply/projects/forms.py

```diff
@@ -381,10 +381,10 @@ def __init__(self, user=None, *args, **kwargs):
 
 class ReportEditForm(forms.ModelForm):
     public_content = RichTextField(
-        help_text="This section will be shared with the wider community."
+        help_text="This section of the report will be shared with the broader OTF community."
     )
     private_content = RichTextField(
-        help_text="This section will be shared with staff members only."
+        help_text="This section of the report will be shared with OTF staff only."
```

Comment:
we want to avoid using the term "OTF" as we need to consider other organisations. Can you remove it please.

Reply:
ah fair, was just matching the wording in the designs, will swap put